astroquery/esa/hubble/core.py

```diff
@@ -564,19 +600,12 @@ def query_hst_tap(self, query, async_job=False, output_file=None,
         -------
         A table object
         """
-        if async_job:
-            job = self._tap.launch_job_async(query=query,
-                                             output_file=output_file,
-                                             output_format=output_format,
-                                             verbose=verbose,
-                                             dump_to_file=output_file is not None)
-        else:
-            job = self._tap.launch_job(query=query, output_file=output_file,
-                                       output_format=output_format,
-                                       verbose=verbose,
-                                       dump_to_file=output_file is not None)
-        table = job.get_results()
-        return table
+        warnings.warn(
```

Comment:
Doing it with the deprecated decorator from astropy.utils may be a tiny bit simpler, but at this point you don't need to change this.

Reply:
I have also included the decorator, but I left the warning.

Reply:
`@deprecated` ensures that a deprecation warning is emitted, so the warnings raised by the decorated function itself are duplicates.

Reply:
Done, commented below.